examples/customresourceinterpreter/webhook/app/workloadwebhook.go

```diff
@@ -86,3 +88,21 @@ func (e *workloadInterpreter) responseWithExploreRetaining(desiredWorkload *work
 	}
 	return interpreter.PatchResponseFromRaw(req.Object.Raw, marshaledBytes)
 }
+
+func (e *workloadInterpreter) responseWithExploreAggregateStatus(workload *workloadv1alpha1.Workload, req interpreter.Request) interpreter.Response {
+	wantedWorkload := workload.DeepCopy()
+	var readyReplicas int32
+	for _, item := range req.AggregatedStatus {
+		// NOTE: this is just an example, you need to change the method of updating status according to the actual situation
+		if !item.Applied {
+			continue
+		}
+		readyReplicas += 1
```

Comment:
Why not sum the `.status.readyReplicas`? 
We can add a comment here to show how we aggregate the status for the example `Workload`, but this is `not an example`, as we'll add E2E tests for this.

Reply:
It seems karmada didn't collect status info for the example workload resource, I didn't see any `readyReplicas` data from resource bindings.

Reply:
You are right, we don't have the `example controller` to update the status, we simulate the controller behavior in E2E tests.

Reply:
So do I need to update this now? Maybe I can update the collection method when I add the E2E test, so I can debug if the method is correct.

Reply:
I think we should do it in this PR as we can confirm if all the information passed to webhook is enough.

Reply:
I can update the code, but I can barely test it because I can't use kubectl to update the status of the workload object or the resource binding object, and I can't simulate the situation where the status of the workload object is updated.

Reply:
That's true. We can test it in E2E.